spec/controllers/admin/categories_controller_spec.rb

```diff
@@ -16,6 +16,13 @@
     assert_response :redirect, :action => 'index'
   end
 
+  it "test_new" do
```

Comment:
Wouldn't the error here just be the page not loading? The only way the 'new' action is being hit is by clicking that one button that leads to this page. There's not the possibility of it not saving and throwing an error. I think.